scripts/run.sh

```diff
@@ -64,7 +64,10 @@ echo "Test pod exited with code:${podExitCode}"
 
 #Collect artifacts
 retrive-artifacts
+retrieveArtifactsExitCode=$?
+
 
 #Clean up
 delete_lease
-exit ${podExitCode}
+exitCode=$(${podExitCode}+${retrieveArtifactsExitCode})
```

Comment:
what does this do? I don't think I've ever seen an addition operator used in bash